css/ampshared.css

```diff
@@ -272,6 +272,14 @@ i-amphtml-sizer {
   display: none;
 }
 
+/** Hide all text node children of non-container elements. */
+.i-amphtml-notbuilt:not(.i-amphtml-layout-container) ,
+[layout]:not([layout=container]):not(.i-amphtml-element)
```

Comment:
This should also include the implicit responsive selectors from above:

```css
[width][height][sizes]:not([layout]):not(.i-amphtml-element),
[width][height][heights]:not([layout]):not(.i-amphtml-element)

Reply:
Correct. All selectors from above minus `> *` should be here as well.